dev/src/generate/mod.rs

```diff
@@ -30,6 +31,9 @@ pub fn run(language: &str) -> Result<()> {
 
     match language {
         "python" | "py" => binding_python::generate(project_root, &services),
+        "nodejs" | "typescript" | "js" | "ts" => {
```

Comment:
Apologies for using the incorrect terminology `language`, which may have caused some confusion. We are referring to the name of the bindings, and we do not plan to build `opendal-typescript`. Therefore, please use `nodejs` or `js` instead.